docs/src/introduction/standard_library.md

```diff
@@ -0,0 +1,15 @@
+# Standard Library
+
+Similar to Rust, Sway comes with its own standard library.
+
+The Sway Standard Library is the foundation of portable Sway software, a set of minimal shared abstractions for the broader Sway ecosystem. It offers core types, like `Result<T, E>` and `Option<T>`, library-defined operations on language primitives, native asset management, blockchain contextual operations, access control and storage management, among many other things.
+
+The standard library is made implicitly available to all Forc projects created using [`forc init`](../forc/commands/forc_init.md). Importing items from the standard library can be done using the `use` keyword. Example:
+
+```sway
+use std::address::Address;
+
```

Comment:
So GitHub is not letting me suggest adding the three backticks at the end of this code block. They seem to have been removed from my earlier suggestion as well. Looks like this happens because "suggestions" themselves also end with three backticks and so somehow the editor merge the two sets of backticks.

Reply:
How can I help to resolve this?